examples/baremetal-tinkerbell-machinedeployment.yaml

```diff
@@ -52,61 +52,15 @@ spec:
                   name: machine-controller-baremetal-tb
                   key: token
             driverSpec:
-              provisionerIPAddress: << PROVISIONER_IP_ADDRESS >>
-              mirrorHost: << MIRROR_HOST >>
-              hardware:
-                id: << MACHINE_NAME >>
-                metadata:
-                  facility:
-                    facilitycode: << FACILITY_CODE >>
-                    planslug: << PLAN_SLUG >>
-                  state: ""
-                  instance:
-                    operatingsystemversion:
-                      distro: << OS_NAME >>
-                      imagetag: << IMAGE_TAG >>
-                      osslug: << OS_NAME >>
-                      slug: << OS_NAME >>
-                      version: << OS_VERSION >>
-                    storage:
-                      disks:
-                        - device: /dev/sda
-                          wipetable: true
-                          partitions:
-                            - size: 4096
-                              label: BIOS
-                              number: 1
-                            - size: 3993600
-                              label: SWAP
-                              number: 2
-                            - size: 0
-                              label: ROOT
-                              number: 3
-                      filesystems:
-                        - mount:
-                            point: /
-                            create:
-                              options:
-                                - -L
-                                - ROOT
-                            device: /dev/sda3
-                            format: ext4
-                        - mount:
-                            point: none
-                            create:
-                              options:
-                                - -L
-                                - SWAP
-                            device: /dev/sda2
-                            format: swap
-                network:
-                  interfaces:
-                    - dhcp:
-                        arch: x86_64
-                        uefi: false
-                      netboot:
-                        allowpxe: false
-                        allowworkflow: false
+              clusterName: "<< CLUSTER_NAME >>"
+              osImageUrl: "<< OS_IMAGE_URL >>"
+              hegelUrl: "<< HEGEL_URL >>"
```

Comment:
This won't be needed anymore since we are gonna create and pass the cloud-init as part of the workflow.